datasets/comonscreens.yaml

```diff
@@ -0,0 +1,31 @@
+Name: Common Screens
+Description: A corpus of web screenshot and metadata data composed of over 70 million websites.
+Documentation: https://commonscreens.com/?page_id=1492
+Contact: admin@commonscreens.com
+ManagedBy: "[Common Screens](https://commonscreens.com/)"
+UpdateFrequency: Monthly
+Tags:
+  - aws-pds
+  - encyclopedic
+  - natural language processing
+  - internet
+License: This data is available for anyone to use subject to rules and regulations of USA.

```

Comment:
Is there a more defined license possible? Something like a Creative Commons license or similar?

Reply:
changed license